doc/source/train/examples/lightning/lightning_mnist_example.ipynb

```diff
@@ -0,0 +1,532 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Finetuning a Pytorch Lightning Image Classifier\n",
+    "\n",
+    "This example introduces how to train a Pytorch Lightning Module using AIR {class}`LightningTrainer <ray.train.lightning.LightningTrainer>`. We will demonstrate how to train a basic neural network on the MNIST dataset with distributed data parallelism.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import torch\n",
+    "import torch.nn as nn\n",
+    "import torch.nn.functional as F\n",
+    "from torch.utils.data import DataLoader, random_split, Subset\n",
+    "from torchmetrics import Accuracy\n",
+    "from torchvision.datasets import MNIST\n",
+    "from torchvision import transforms\n",
+    "\n",
+    "import pytorch_lightning as pl\n",
+    "from pytorch_lightning import trainer\n",
+    "from pytorch_lightning.core import datamodule\n",
+    "from pytorch_lightning.loggers.csv_logs import CSVLogger"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare Dataset and Module\n",
+    "\n",
+    "The Pytorch Lightning Trainer takes either `torch.utils.data.DataLoader` or `pl.LightningDataModule` as data inputs. You can keep using them without any changes for the Ray AIR LightningTrainer. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class MNISTDataModule(pl.LightningDataModule):\n",
+    "    def __init__(self, batch_size=100):\n",
+    "        super().__init__()\n",
+    "        self.data_dir = os.getcwd()\n",
+    "        self.batch_size = batch_size\n",
+    "        self.transform = transforms.Compose(\n",
+    "            [transforms.ToTensor(), transforms.Normalize((0.1307,), (0.3081,))]\n",
```

Comment:
Btw, is the batch size of a datamodule global batch size across all workers? Or batch size per worker?

What about for a torch DataLoader?

Reply:
Both datamodule and dataloader use a global batch size. PTL will create a distributed sampler for each worker.